vcr/oidc4vci/service/identifiers.go

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (C) 2023 Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package service
```

Comment:
rather generic package name.
Can't it be placed one lever higher, under `oidc4vci`?

Reply:
I wanted to put the rest under a new `client` package but didn't want to do that in this PR (to cater a sort of generic OpenID4VCI library);

- `openid4vci/client`
- `openid4vci/services`